ethers-providers/src/middleware.rs

```diff
@@ -719,6 +719,19 @@ pub trait Middleware: Sync + Send + Debug {
         self.inner().txpool_status().await.map_err(MiddlewareError::from_err)
     }
 
+    /// Simulates a set of bundles
+    /// Implementation:
+    /// Ref:
+    /// [Here](https://github.com/ledgerwatch/erigon/issues/4471)
+    async fn eth_call_many<T: Into<TypedTransaction> + Send + Sync>(
+        &self,
+        req: Vec<EthCallManyBundle<T>>,
+        state_diff: Option<HashMap<H160, EthCallManyBalanceDiff>>,
+        block: Option<BlockNumber>,
```

Comment:
I think the order is wrong

https://github.com/ledgerwatch/erigon/blob/513fd50fa501ab6385dc3f58b18079d806d6ff5a/turbo/jsonrpc/eth_callMany.go#L72-L72